receiver/jmxreceiver/config.go

```diff
@@ -34,23 +34,17 @@ type config struct {
 	GroovyScript string `mapstructure:"groovy_script"`
 	// The duration in between groovy script invocations and metric exports (10 seconds by default).
 	// Will be converted to milliseconds.
-	Interval time.Duration `mapstructure:"interval"`
+	CollectionInterval time.Duration `mapstructure:"collection_interval"`
 	// The JMX username
 	Username string `mapstructure:"username"`
 	// The JMX password
 	Password string `mapstructure:"password"`
-	// The metric exporter to use ("otlp" or "prometheus", "otlp" by default).
-	Exporter string `mapstructure:"exporter"`
-	// The OTLP Receiver endpoint to send metrics to ("localhost:55680" by default).
+	// The OTLP Receiver endpoint to send metrics to ("0.0.0.0:<random open port>" by default).
 	OTLPEndpoint string `mapstructure:"otlp_endpoint"`
 	// The OTLP exporter timeout (5 seconds by default).  Will be converted to milliseconds.
 	OTLPTimeout time.Duration `mapstructure:"otlp_timeout"`
 	// The headers to include in OTLP metric submission requests.
 	OTLPHeaders map[string]string `mapstructure:"otlp_headers"`
```

Comment:
These all map to the java properties, and only the `OTLPEndpoint` has any bearing on the underlying OTLP receiver's config (gRPC endpoint): https://github.com/open-telemetry/opentelemetry-java-contrib/blob/master/contrib/jmx-metrics/README.md#configuration

Reply:
I meant instead of OLTP_XYZ just embed the entire otlpexporter.Config which has all the fields

Reply:
My mistake, I assumed you were referring to the receiver config since that's whats being stood up.  There's been a request from @jrcamp to add `endpoint` that translates to service url for the receiver creator in a subsequent PR and I think that would conflict w/ using the otlp exporter config directly.

Updated to embed from timeout settings and renamed "headers" if that works for you.

Reply:
Instead of embedding at the top level struct could you have it as a struct member to avoid name conflicts? ie,

```go
type config struct {
	configmodels.ReceiverSettings `mapstructure:",squash"`
	// The path for the JMX Metric Gatherer uber JAR (/opt/opentelemetry-java-contrib-jmx-metrics.jar by default).
	JARPath string `mapstructure:"jar_path"`
        ...
        OTLPConfig otlp.Config `mapstructure:"otlp_config"`
}
```

Reply:
@jrcamp I considered this but don't see an immediate advantage in doing so for these three fields, which again are proxied to the java metric client and not for an actual collector otlp exporter (so most of GRPCClientSettings, QueueSettings, RetrySettings, and ExporterSettings have no significance).

Reply:
I understand now, but still better to be consistent and use the same field names so a copy paste can work. See the latest comment from me.